### PR TITLE
setup-focused getting started walkthrough

### PR DESCRIPTION
**Pull request by Eskibear**
for insider users to dogfood this feature

### CODE REVIEW

**Comment by jdneo** [comment]
How about the AB testing, should we stop it?

**Comment by Eskibear** [comment]
No, it doesn't matter with current welcome page implemented in Java pack.

File: package.json [hunk 1]
```diff
@@ -52,6 +52,89 @@
...
+            "media": {
+              "path": "walkthrough/open-project.png",
+              "altText": "Open Project Folder"
+            },
+            "when": "isMac && workspaceFolderCount == 0"
+          },
+          {
+            "id": "java.showProjectExplorer",
+            "title": "Explore your project",
+            "description": "View class paths and dependencies, and create new projects, packages, and classes.\n[Show Java Projects](command:javaProjectExplorer.focus)",
```
Comment (jdneo) [inline package.json:102]: typo: class path -> classpath

Comment (jdneo) [inline package.json:102]: How to handle the case that the extension is not activated? (command `javaProjectExplorer.focus` not exists)

Adding a when clause?

**Comment by Eskibear** [comment]
> How about the AB testing, should we stop it?

It's independent with this Getting-Started walkthrough.

File: package.json [hunk 2]
```diff
@@ -89,7 +89,7 @@
           {
             "id": "java.openProjectFolder.mac",
             "title": "Open project folder",
-            "description": "Open an existing folder that contains your Java project, or create a new folder.\n[Open Folder...](command:workbench.action.files.openFileFolder)\n[Create new Project](command:java.project.create)",
+            "description": "Open an existing folder that contains your Java project, or create a new folder.\n[Open Folder...](command:workbench.action.files.openFileFolder)\n\n[Create new Project](command:java.project.create)",
```
Comment (jdneo) [inline package.json]: Is the two `\n` on purpose?

**Pull request merged**